package.json

```diff
@@ -14,7 +14,6 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^5.0.1",
-    "react-router-dom": "^6.22.0",
```

Comment:
Does this needs to be deleted for your changes?

Reply:
yes